src/vf2symbols/write_symbol_from_svgs.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generates an Apple custom symbol using SVGs.
+
+Fills an apple symbols variants with the provided SVGs.
+Takes one or more space seperated pairs of symbol name and its SVG filepath.
+
+https://developer.apple.com/documentation/uikit/uiimage/creating_custom_symbol_images_for_your_app#3369885
+
+Usage:
+write_symbol_from_svgs.py -out [output path] [symbol_layer_name=svg_path ....]
+
+Example:
+write_symbol_from_svgs.py -out apple_symbol.svg Regular-M=regular.svg Regular-L=large.svg
+"""
+import re
+import sys
+
+from absl import app
+from absl import flags
+
+from fontTools import svgLib
+from fontTools.pens.svgPathPen import SVGPathPen
+from picosvg.geometric_types import Rect
+from picosvg.svg import SVG
+from vf2symbols.symbol import Symbol
+
+FLAGS = flags.FLAGS
+
+# internal flags, typically client wouldn't change
+flags.DEFINE_string("out", None, "Output file.")
+
+
+def parse_float(string):
+    return float(re.compile("\d+([.]\d*)?").match(string).group(0))
```

Comment:
why can't you just pass the string straight to float()? - I don't quite follow the purpose of this function

Reply:
to discard the units ex:"48.0px"